### PR TITLE
init

### PR DESCRIPTION
**Pull request by xverizex**
### Please confirm your submission meets all the criteria

- [x] I have read the [App Requirements][reqs] and [App Maintenance][maint] pages.
- [x] My pull request follows the instructions at [App Submission][submission].
- [x] I am using only the minimal set of permissions. *(If not, please explain each non-standard permission.)*
- [x] All assets referenced in the manifest are redistributable by any party.  If not, the unredistributable parts are using an extra-data source type.
- [x] I am an upstream contributor to the project. If not, I contacted upstream developers about submitting their software to Flathub. **Link:**
- [x] I own the domain used in the application ID or the domain has a policy for delegating subdomains (e.g. GitHub, SourceForge).
- [x] Any additional patches or files have been submitted to the upstream projects concerned. *(If not, explain why.)*

[reqs]: https://github.com/flathub/flathub/wiki/App-Requirements
[maint]: https://github.com/flathub/flathub/wiki/App-Maintenance
[submission]: https://github.com/flathub/flathub/wiki/App-Submission

### CODE REVIEW

**Comment by xverizex** [comment]
I use permission=home for storage crypto keys.
I use permission=pulseaudio for sound notifications.

**Comment by hfiguiere** [comment]
you shouldn't have your source code here. Just the manifest.

File: org.xverizex.nem-desktop.json [hunk 1]
```diff
@@ -0,0 +1,41 @@
+{
+    "app-id" : "org.xverizex.nem-desktop",
+    "runtime" : "org.gnome.Platform",
+    "runtime-version" : "40",
```
Comment (hfiguiere) [inline org.xverizex.nem-desktop.json:4]:
```suggestion
    "runtime-version" : "41",
```

File: org.xverizex.nem-desktop.json [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+    "sdk" : "org.gnome.Sdk",
+    "command" : "nem",
+    "finish-args" : [
+        "--share=network",
+        "--share=ipc",
+        "--socket=fallback-x11",
+        "--socket=wayland",
+        "--socket=pulseaudio",
+        "--socket=session-bus",
```
Comment (hfiguiere) [inline org.xverizex.nem-desktop.json:13]:
```suggestion
```

Add whichever name you need instead

File: org.xverizex.nem-desktop.json [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+        "--filesystem=home:create",
+        "--talk-name=org.xverizex.nem-desktop"
```
Comment (hfiguiere) [inline org.xverizex.nem-desktop.json:15]:
```suggestion
        "--filesystem=home:create"
```

this talk name is implied (it's the app id). As for the `:create` on `home`, I don't think so. Please restrict these permissions.

File: org.xverizex.nem-desktop.json [hunk 4]
```diff
@@ -0,0 +1,41 @@
...
+    ],
+    "modules" : [
+        {
+            "name" : "nem",
+            "builddir" : true,
+            "buildsystem" : "meson",
+            "sources" : [
+                {
+                    "type" : "git",
+                    "url" : "https://github.com/xverizex/flathub/tree/org.xverizex.nem-desktop"
```
Comment (hfiguiere) [inline org.xverizex.nem-desktop.json:36]: You need a specific commit / tag.

File: org.xverizex.nem-desktop.metainfo.xml [hunk 5]
```diff
@@ -0,0 +1,28 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<component type="desktop">
+  <id>org.xverizex.nem-desktop</id>
+  <name>Nem Desktop</name>
+  <developer_name>Dmitrii Naidolinskii</developer_name>
+  <metadata_license>CC0-1.0</metadata_license>
+  <project_license>GNU</project_license>
```
Comment (hfiguiere) [inline org.xverizex.nem-desktop.metainfo.xml:7]: this is an invlalide SPDX license identified. see https://spdx.org/licenses/

**Review by hfiguiere — CHANGES_REQUESTED** [review]
as mentionned the source code of you app shouldn't be in there.

**Comment by hfiguiere** [comment]
also the app-id doesn't seem to be valid as the domain doesn't seem to exist.

**Pull request closed without merging**